tests/ha/test_server_pod_failure.py

```diff
@@ -1612,7 +1551,7 @@ def test_copy_obj_safe_server_pod_shutdown(self):
         LOGGER.info("Step 2: Shutdown the server pod safely by making replicas=0")
         LOGGER.info("Get pod name to be shutdown")
```

Comment:
Don't we need to use lib in these tests also?

Reply:
> Don't we need to use lib in these tests also?

Will take care test change in next PR. This one is just for lib change